lib/jsonb_accessor/macro.rb

```diff
@@ -44,19 +44,21 @@ def jsonb_accessor(jsonb_attribute, field_types)
           jsonb_values = public_send(jsonb_attribute) || {}
           jsonb_values.each do |store_key, value|
             name = names_and_store_keys.key(store_key)
-            write_attribute(name, value)
+            write_attribute(name, value) if name.present?
```

Comment:
`present?` could trip us up in the future. For instance, if someone has an array field that is `nil` by default but is eventually set to `[]`